rg-manifest.rc

```diff
@@ -0,0 +1,2 @@
+#define RT_MANIFEST 24
+1 RT_MANIFEST "rg.exe.manifest"
```

Comment:
I can't comment on it, but where did the `rg.exe.manifest` file come from? How was it generated?

Also, I think it would be better to put these files into a `windows` sub-directory, instead of polluting the root of the repo even more than it is.

Reply:
Oh I see, GitHub falsely detects it as a binary file, but it's just plain text. Phew. That's good. Yeah, I think just move these into a new `windows` sub-directory. Thanks!